crates/bin/pcli/src/command/query/shielded_pool.rs

```diff
@@ -13,16 +12,6 @@ pub enum ShieldedPool {
         /// The height to query.
         height: u64,
     },
-    /// Queries the state commitment tree's block anchor for a given height.
-    BlockAnchor {
-        /// The height to query.
-        height: u64,
-    },
-    /// Queries the state commitment tree's epoch anchor for a given epoch index.
-    EpochAnchor {
-        /// The epoch to query.
-        epoch: u64,
-    },
```

Comment:
This is functionality that should really be part of an indexer / explorer, rather than asking every full node to keep the data around in case someone wants to run this command against a random fullnode.